recipes/libpng/all/conanfile.py

```diff
@@ -126,25 +126,21 @@ def generate(self):
     def _patch_sources(self):
         apply_conandata_patches(self)
         if self.settings.os == "Windows":
-            if Version(self.version) <= "1.5.2":
-                replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
-                                      'set(PNG_LIB_NAME_STATIC ${PNG_LIB_NAME}_static)',
-                                      'set(PNG_LIB_NAME_STATIC ${PNG_LIB_NAME})')
-            else:
-                replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
-                                    'OUTPUT_NAME "${PNG_LIB_NAME}_static',
-                                    'OUTPUT_NAME "${PNG_LIB_NAME}')
```

Comment:
it's a bad idea to remove this patch, I don't understand why it has been accepted.